packaging/torchtext/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: torchtext
+  version: "{{ environ.get('BUILD_VERSION') }}"
+
+source:
+  path: "{{ environ.get('SOURCE_ROOT_DIR') }}"
+
+requirements:
+  build:
+    - {{ compiler('c') }} # [win]
+    - {{ compiler('cxx') }} # [win]
+    - cmake
+
+  host:
+    - python
+    - setuptools
+    - cpuonly
+    {{ environ.get('CONDA_PYTORCH_BUILD_CONSTRAINT') }}
+
+  run:
+    - python
+    - requests
+    - tqdm
+    {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
+
+build:
+  string: py{{py}}
+
+test:
+  imports:
+    - torchtext
```

Comment:
@zhangguanheng66 This is where the import test is defined.

Reply:
This is the old conda build script with some import tests [link](https://github.com/pytorch/text/blob/32bc374d7d8b15ed431dfaa64304a974a959ea5e/build_tools/conda/torchtext/meta.yaml#L25)

Reply:
Oh interesting. why was it removed?